packages/vscode-extension/package.nls.json

```diff
@@ -24,7 +24,7 @@
     "teamstoolkit.appStudioLogin.loginCancel": "Sign-in canceled. Teams Toolkit needs a Microsoft 365 account with custom app upload permission. If you're a Visual Studio subscriber, create a developer sandbox with the Microsoft 365 Developer Program (https://developer.microsoft.com/en-us/microsoft-365/dev-program).",
     "teamstoolkit.appStudioLogin.message": "Teams Toolkit needs a Microsoft 365 account with custom app upload permission. If you're a Visual Studio subscriber, create a developer sandbox with the Microsoft 365 Developer Program.",
     "teamstoolkit.azureLogin.failToFindSubscription": "We couldn't find a subscription.",
-    "teamstoolkit.azureLogin.message": "The Teams Toolkit requires an Azure account and subscription to deploy the Azure resources for your project. You won't be charged until you confirm.",
+    "teamstoolkit.azureLogin.message": "The Teams Toolkit will use Microsoft authentication to sign in Azure account and subscription to deploy the Azure resources for your project. You will not be charged without further confirmation.",
```

Comment:
I'd propose to change the second line to the existing one i.e. You won't be charged until you confirm.
Logic: We use contractions i.e. won't, can't, you've, etc.as per our style guide. And, the existing line is easier to perceive, has a personalized tone, and clear. The revised line doesn't convey clearly - without whose further confirmation?